src/alignof.cc

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
Is there a specific reason why this is in a new file rather than [`src/node_os.cc`](https://github.com/nodejs/io.js/blob/2b1c01c2cc203dd6131c3fc6238cf62a2e697f63/src/node_os.cc)?